src/index.js

```diff
@@ -1 +1,18 @@
 import './style.css';
+import displayMovies from './modules/displayMovies.js';
+
+window.addEventListener('load', () => {
+  displayMovies();
+  // const mainContainer = document.getElementById('mainContainer');
+
+  //   const nextSlide = () => {
+  //     mainContainer.scrollLeft -= 700;
+  //   };
+
+  //   const prevSlide = () => {
+  //     mainContainer.scrollLeft += 700;
+  //   };
+
+//   prevBtn.addEventListener('click', nextSlide);
+//   nextBtn.addEventListener('click', prevSlide);
```

Comment:
Kindly remove these comments as it is against best practices and extra code as comments make our code messy. They can also cause some conflicts in the future. 😊

Reply:
I have done the changes you mentioned. Thanks for the kind feedback.